wiki-fe/src/app/_components/Footer.jsx

```diff
@@ -5,7 +5,7 @@ export default function Footer() {
   return (
     <div className="footer-content">
         <p>&copy; 2024 WikiDengue. Todos os direitos reservados.</p>
-        <p><a href="{% url 'encyclopedia:random' %}">Contato</a> | <a href="{% url 'encyclopedia:random' %}">Sobre</a></p>
+        <p><a href="{% url 'encyclopedia:random' %}">Contato</a> | <a href="{% url 'encyclopedia:random' %}">Sobre</a> | <a href="{% url 'encyclopedia:random' %}">Políticas de Privacidade</a></p>
```

Comment:
Isso aqui não vai dar mais certo porque isso aí é sintaxe do django, pra criar menus de nagevação tem uma propriedade específica do next js:

https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating